x-pack/plugins/elastic_assistant/server/lib/langchain/graphs/default_assistant_graph/graph.ts

```diff
@@ -95,119 +83,86 @@ export const getDefaultAssistantGraph = ({
         value: (x: string, y?: string) => y ?? x,
         default: () => '',
       },
+      llmType: {
+        value: (x: string, y?: string) => y ?? x,
+        default: () => 'unknown',
+      },
+      bedrockChatEnabled: {
+        value: (x: boolean, y?: boolean) => y ?? x,
+        default: () => false,
+      },
+      isStream: {
+        value: (x: boolean, y?: boolean) => y ?? x,
+        default: () => false,
+      },
       conversation: {
         value: (x: ConversationResponse | undefined, y?: ConversationResponse | undefined) =>
           y ?? x,
         default: () => undefined,
       },
+      conversationId: {
+        value: (x: string, y?: string) => y ?? x,
+        default: () => '',
+      },
+      responseLanguage: {
+        value: (x: string, y?: string) => y ?? x,
+        default: () => '',
```

Comment:
```suggestion
        default: () => 'English',
```

Defaulting this to an empty string will break our titling prompt which relies on this definition to be a language

Reply:
[Empty string for language, broken title](https://smith.langchain.com/public/18d71518-8c8e-492b-b67d-1d27413bd849/r)
[English for language, proper title](https://smith.langchain.com/public/3e4ab6c1-f476-4dfb-aa84-69f062ebabfa/r)

Reply:
Or maybe it should be 
```suggestion
        default: () => responseLanguage,
```

Notice `responseLanguage` is still on `GetDefaultAssistantGraphParams` but is unused. Should `responseLanguage` be moved to the `inputs` for `state`?

Reply:
Sure it can be moved to the invoke. Any parameter that is serializable could be moved, seems like I did miss this one.